frontend/src/components/dashboard/create/Create.tsx

```diff
@@ -23,59 +18,39 @@ const Create = () => {
   const { id } = useParams()
   const history = useHistory()
 
-  const [campaign, setCampaign] = useState(new Campaign({}))
+  const { campaign, setCampaign } = useContext(CampaignContext)
+  const {
+    handleFinishLater: finishLaterContextHandler,
+    finishLaterContent,
+  } = useContext(FinishLaterModalContext)
   const [isLoading, setLoading] = useState(true)
-  const finishLaterCallbackRef: React.MutableRefObject<
-    (() => void) | undefined
-  > = useRef()
-
-  async function loadProject(id: string) {
-    const campaign = await getCampaignDetails(+id)
-    setCampaign(campaign)
-    setLoading(false)
-  }
 
   useEffect(() => {
     if (!id) return
+    async function loadProject(id: string) {
+      const campaign = await getCampaignDetails(+id)
+      setCampaign(campaign)
+      setLoading(false)
+    }
     loadProject(id)
-  }, [id])
+  }, [id, setCampaign])
 
   async function handleFinishLater() {
-    if (campaign.status === Status.Draft) {
+    if (campaign.status === Status.Draft && finishLaterContent) {
```

Comment:
Minor point. Behaviour for this is slightly changed. The GA event is now only sent for pages that trigger the save draft modal. It doesn't fire for the other pages (credentials, send).

Reply:
I'll remove this and merge it in